xml_analyzer/xmlreader.py

```diff
@@ -1,144 +1,153 @@
 import collections
+from typing import List
 import xml.etree.ElementTree as Et
 from .data_container import DataContainer
+from .definitions import AutocalcMethods
 
 
 class XMLReader:
-    autocalc_methods = ['autosum']
 
     def __init__(self, project_file):
```

Comment:
A type hint for project file would be nice. You can indicate if it just the path (a str) or an open file - file object

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/729e6c63dfd4ab70a84ea5ce1e2c3dbe515eb5f1